src/main/java/net/sf/jabref/logic/importer/fetcher/IsbnFetcher.java

```diff
@@ -43,6 +49,9 @@
 
     private static final String URL_PATTERN = "http://www.ebook.de/de/tools/isbn2bibtex?";
 
+    private final ProtectTermsFormatter protectTermsFormatter = new ProtectTermsFormatter(new ProtectedTermsLoader(new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
```

Comment:
What was the reason to reintroduce these formatters? I really don't like them in the fetchers.